### DISCUSSION TITLE
Q: 使用Zeabur部署Waline后出现接口调用与文档描述不一致的情况 || Q: After using Zeabur to deploy Waline, the interface call is inconsistent with the document description.

### DISCUSSION THREAD

**Discussion by nonhana** [walinejs/waline#2362] [Q&A] [answered]
### Describe the bug | 问题描述

如图所示，我直接访问我部署的url，请求的接口路径没有 `/api` 的前缀：
![image](https://github.com/walinejs/waline/assets/105428118/df92385c-7922-4d47-b0a1-76aea0c9de0d)
因此当我在我的hexo博客上部署后，请求的路径发生了错误：
![image](https://github.com/walinejs/waline/assets/105428118/f22a032d-b62a-4a45-a962-1bdddbd4f300)
我想问问这是什么原因导致的，我并没有修改我的zeabur应用，只是把原本的仓库拉下来进行部署

### Website URL | 问题网站

https://greyflowers-waline.zeabur.app

### Where your waline deploy? | 服务部署在哪里？

Zeabur

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

As shown in the figure, I directly access the URL I deployed, and the requested interface path does not have the `/api` prefix:
![image](https://github.com/walinejs/waline/assets/105428118/df92385c-7922-4d47-b0a1-76aea0c9de0d)
So when I deployed it on my hexo blog, an error occurred in the requested path:
![image](https://github.com/walinejs/waline/assets/105428118/f22a032d-b62a-4a45-a962-1bdddbd4f300)
I would like to ask what is the cause of this. I did not modify my zeabur application, I just pulled down the original warehouse for deployment.

### Website URL | Problem website

https://greyflowers-waline.zeabur.app

### Where your waline deploy? | Where is the service deployed?

Zeabur

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
再重新部署下，仓库依赖比较老了，刚更新了下


---
Redeploy again. The warehouse dependency is relatively old and has just been updated.
